cdap-ui/app/cdap/services/global-constants.js

```diff
@@ -203,7 +203,9 @@ const GLOBALS = {
       pluginMissingError: 'Please choose a plugin.',
       pluginVersionMissingError: 'Please choose artifact version for the plugin.'
     }
-  }
+  },
+
+  NUMBER_TYPES
```

Comment:
If you're already exporting `NUMBER_TYPES` directly then you don't need to make it part of `GLOBALS` I guess.